storage-proto/proto/solana.storage.transaction_by_addr.rs

```diff
@@ -120,4 +120,5 @@ pub enum InstructionErrorType {
     InvalidAccountOwner = 46,
     ArithmeticOverflow = 47,
     UnsupportedSysvar = 48,
+    IllegalOwner = 49,
```

Comment:
The files in `/proto` are automatically generated from the .proto files in `/src`, so the next time someone runs that build and commits it, this will be overwritten. I'll add a readme and some comments to the crate, but got any ideas for stronger ways of communicating to not edit these files manually?